index.bs

```diff
@@ -3971,6 +4087,33 @@ Note: there's a 1:1 relationship between the <code>script.RealmInfo</code>
 
 The <code>script.RealmInfo</code> type represents the properties of a realm.
 
+<div algorithm>
+To <dfn>get the top level window</dfn> of the given |realm|:
+
+1. Let |global object| be the [=realm/global object=] of the |realm|.
+
+1. While |global object| is a {{Window}} object and has a non-null [=parent=], set
```

Comment:
This reference to `[=parent=]` is wrong because it's for a navigable's parent, which isn't a property of a `{{Window}}` object. But also I don't know why you're trying to get the top-level window here? iframes have their own browsing context, so I think something in the model is wrong.

Reply:
Yeah, it was added before the change in the Classic, Fixed.